cardano-wallet.cabal

```diff
@@ -79,6 +87,26 @@ executable cardano-wallet-server
       Main.hs
 
 
+executable cardano-generate-mnemonic
+  default-language:
+      Haskell2010
+  default-extensions:
+      NoImplicitPrelude
+      OverloadedStrings
+  ghc-options:
+      -threaded -rtsopts
+      -with-rtsopts=-N
+      -Wall
+  build-depends:
+         base
+       , cardano-wallet
+       , aeson
+  hs-source-dirs:
+      src/Cardano/Wallet/Mnemonic
```

Comment:
All executables should be located in `app/<executable-sub-folder>`. Let's try to stick with that :+1:

Reply:
added the executable to `app/mnemonic`